src/main/java/io/anserini/qa/passage/IdfPassageScorer.java

```diff
@@ -101,14 +118,19 @@ public void score(String query, Map<String, Float> sentences) throws Exception {
           TermQuery q = (TermQuery) qp.parse(term);
           Term t = q.getTerm();
           double termIDF = similarity.idf(reader.docFreq(t), reader.numDocs());
-          termIdfMap.put(term, String.valueOf(termIDF));
 
           if (questionTerms.contains(t.toString()) && !seenTerms.contains(t.toString())) {
```

Comment:
But, we decided to do this based only on the quality of our answers in the demo and we didn't measure the effect quantitatively. I will look into separately.